### PR TITLE
Dark theme originally made by SKIN-FACTORY

### PR DESCRIPTION
**Pull request by laurenz1337**
I updated all offline image/font sources and updated/changed a lot of values too
[Source](https://userstyles.org/styles/110257/wa-web-dark-skin-v-3-0-by-skin-factory)

### CODE REVIEW

**Comment by laurenz1337** [comment]
When you are going to implement the theme, you might want to download the linked images and load them locally so they still appear, if the source goes offline for some reason.

**Comment by Aluxian** [comment]
Yep, good idea! Its also faster.

--
Alexandru Rosianu Full-Stack Developer aluxian.com | startersquad.com On Mon, May 23, 2016 at 2:29 PM, Laurenz < notifications@github.com [notifications@github.com] > wrote:
When you are going to implement the theme, you might want to download the linked images and load them locally so they still appear, if the source goes offline for some reason.

File: src/themes/Shadow.css [hunk 1]
```diff
@@ -0,0 +1,682 @@
...
+	background-color: #323232 !important;
+	color: #898989;
+}
+
+.list-action {
+	border-bottom: 1px solid #323232;
+}
+
+.bubble-attach .preview-vcard.no-image, .icon-broadcast-default, .icon-group-default, .icon-user-default, .icon-user-default-square {
+	background-image: url(https://drive.google.com/uc?export=view&id=0B03MSLYuBVBVejlmaGp2Z3cxYkk);
```
Comment (laurenz1337) [inline src/themes/Shadow.css]: Image for users with no avatar image

File: src/themes/Shadow.css [hunk 2]
```diff
@@ -0,0 +1,682 @@
+/* @source https://userstyles.org/styles/110257/wa-web-dark-skin-v-3-0-by-skin-factory */
+/* Modified by Laurenz1337 */
+
+@font-face {
+	font-family: 'Roboto Light' !important;
+	font-style: normal !important;
+	font-weight: 400 !important;
+	src: local('Roboto Light'), local('RobotoLight'), url(https://cdn.rawgit.com/google/fonts/master/apache/roboto/Roboto-Light.ttf) format('ttf') !important;
```
Comment (laurenz1337) [inline src/themes/Shadow.css]: Loads the roboto font from the Google Fonts git hub repository

File: src/themes/Shadow.css [hunk 3]
```diff
@@ -426,6 +426,10 @@ select {
 	font-size: 12px !important;
 }
 
+.intro-image {
+	background-image: url('http://i.imgur.com/4UUlItJ.png');
```
Comment (laurenz1337) [inline src/themes/Shadow.css]: Edited the intro image to have a transparent background

File: src/themes/Shadow.css [hunk 4]
```diff
@@ -679,4 +683,8 @@ a.button.button-contact {
 
 .hidden-token > .emojitext {
 	color: white;
-}
\ No newline at end of file
+}
+
+.message > button {
```
Comment (laurenz1337) [inline src/themes/Shadow.css]: Makes the "Messages are now encrypted" message more readable.

**Comment by Aluxian** [comment]
Thanks!

**Pull request merged**